tests/test_gpg.py

```diff
@@ -52,7 +61,11 @@ def test_sd_gpg_timeout(self):
         self.assertFileHasLine("/home/user/.profile", line)
 
     def test_we_have_the_key(self):
-        self.assertEqual(get_local_fp(), get_remote_fp())
+        local_fp = get_local_fp()
+        remote_fp = get_remote_fp(expected_fp=local_fp)
+
+        self.assertIsNotNone(local_fp, "Local key not found")
```

Comment:
This covers what I mentioned above but I think it would be better for the assertions to be closer to what they are asserting.